index.html

```diff
@@ -4,314 +4,350 @@
+            <ul>
+              <li>CANOPY</li>
+              <li><img src="./image/image3.png" alt="count1" /></li>
               <li>Back End Dev</li>
-              <li><div class="dot"></div></li>
+              <li><img src="./image/image4.png" alt="count2" /></li>
               <li>2015</li>
             </ul>
-            <p class="pro">
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required.
+            <p>
+              A daily selecion of privately personalized reads; no accounts or
+              sign up required.
             </p>
-            <ul class="language">
-              <li>HTML</li>
-              <li>CSS</li>
-              <li>JavaScript</li>
+            <ul class="lang">
+              <li><h5>html</h5></li>
+              <li><h5>css</h5></li>
+              <li><h5>javascript</h5></li>
             </ul>
-            <a href="#" class="button">See Project</a>
+            <button class="button" type="button">See Project</button>
+          </div>
+        </div>
+
+        <div class="card4">
+          <div class="card4001">
+            <img src="./image/card4.png" alt="" />
+          </div>
+          <div class="c002">
+            <h2>Multi-Post Stories</h2>
+            <ul>
+              <li>CANOPY</li>
+              <li><img src="./image/image4.png" alt="count1" /></li>
+              <li>Back End Dev</li>
+              <li><img src="./image/image4.png" alt="count2" /></li>
+              <li>2015</li>
+            </ul>
+            <p>
+              A daily selecion of privately personalized reads; no accounts or
+              sign up required.
+            </p>
+            <ul class="lang">
+              <li><h5>html</h5></li>
+              <li><h5>css</h5></li>
+              <li><h5>javascript</h5></li>
+            </ul>
+            <button class="button" type="button">See Project</button>
           </div>
         </div>
       </section>
-      <!-- ------------------------------------------------about me--------------------------------         -->
-      <section class="border">
-        <section class="about">
-          <h2>
-            About <br />
-            Myself
-          </h2>
-          <p>
-            Hello I'm a software developer! I can help you build a product ,
-            feature or website Look through some of my work and experience! If
-            you like what you see and have a project you need coded, don't
-            hestiate to contact me.
+    </div>
+
+    <section class="about">
+      <div id="about">
+        <section id="about001">
+          <h2 class="my001">About<br />MySelf</h2>
+          <p class="intro">
+            Hello I am a software developer! I can help you build a
+            product,feature or website Look through some of my work and
+            experience! if you like what you see and have a project you need
+            coded, don't hesitate to contact me.
```

Comment:
- [ ] Great work working on the mobile section. However, this _**About Myself**_ section is not identical to that of the Figma template that you chose. Whie you made attempts to make it look identical, the design was not implemented properly. I would advise that you do the following to make it identical:

- Make the div with `id="about"` have a style declaration of `display: flex` and remove the style declaration `grid-template-columns: 1fr 1.7fr;`
- Remove the `width` declaration for the element with `id="about001"`
- Reduce the `margin-top` of the `button` element with text `Get My Resume` to like `1rem`

#### The above is not the perfect implementation, but it would make your design look identical to the Figma template that you chose. You don't have to follow my suggestion word for word, but it is required that you make your design identical to the Figma template. 👍 


### Your Design

![image](https://user-images.githubusercontent.com/77106962/208251936-aa62a465-21b2-49a3-9a4f-2c7106ae08e6.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/208251960-c0a1dcbf-68ec-42ba-93ec-e795d03d8af2.png)